### PR TITLE
Linda scrub 828

### PR DESCRIPTION
**Pull request by Linda-Editor**
Updated some pages to make them version-inclusive/agnostic. I didn't change some references because I think the reference to the version was needed. Let me know if you have any questions.

### CODE REVIEW

File: docs/develop/convert-javascript-to-typescript.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ declare var fabric: any;
             messageBanner = new fabric.MessageBanner(element);
             messageBanner.hideBanner();
             
-            // If not using Excel 2016, use fallback logic.
+            // If not using Excel, use fallback logic.
```
[the remark this answers is not in the document]
Comment (lindalu-MSFT) [reply docs/develop/convert-javascript-to-typescript.md]: @ElizabethSamuel-MSFT Sounds like a safe bet to change to "or later". Makes it more Evergreen.

File: docs/includes/quickstart-vs-prerequisites.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
     > [!NOTE]
     > If you've previously installed Visual Studio 2017, [use the Visual Studio Installer](https://docs.microsoft.com/visualstudio/install/modify-visual-studio) to ensure that the **Office/SharePoint development** workload is installed. 
 
-- Office 2016 
+- Office 
     
     > [!NOTE]
     > If you don't already have Office 2016, you can [register for a free 1-month trial](http://office.microsoft.com/try/?WT%2Eintid1=ODC%5FENUS%5FFX101785584%5FXT104056786).
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/quickstart-vs-prerequisites.md]: This says Office 2016 - maybe "latest version of Office"?

File: docs/publish/centralized-deployment.md [hunk 3]
```diff
@@ -126,20 +126,21 @@ If the add-in supports add-in commands, the commands will appear on the Office a
 
 If the add-in does not support add-in commands, users can add it to their Office application by doing the following:
 
-1.	In Word 2016, Excel 2016, or PowerPoint 2016, choose **Insert** > **My Add-ins**.
+1.	In Word, Excel, or PowerPoint, choose **Insert** > **My Add-ins**.
```
Comment (ElizabethSamuel-MSFT) [inline docs/publish/centralized-deployment.md]: Also, do same instructions apply across the various versions of Office?

**Comment by Linda-Editor** [comment]
Hi Elizabeth, let me know when you're done with your review, and then I'll address your comments. Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
Thanks Linda, but Kim will update.

**Comment by Linda-Editor** [comment]
@ElizabethSamuel-MSFT  Okay, so no action items for me?

**Comment by ElizabethSamuel-MSFT** [comment]
@Linda-Editor No, you're good. Thx.

File: docs/design/task-pane-add-ins.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ Task panes are interface surfaces that typically appear on the right side of the
 
 The following images show the various task pane sizes with the Office ribbon at a 1366x768 resolution. For Excel, additional vertical space is required to accommodate the formula bar.  
 
-*Figure 2. Office 2016 desktop task pane sizes*
+*Figure 2. Office desktop task pane sizes*
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/design/task-pane-add-ins.md]: Fixed.

File: docs/develop/convert-javascript-to-typescript.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/develop/convert-javascript-to-typescript.md]: Fixed.

File: docs/includes/quickstart-vs-prerequisites.md (hunk 2 above)
Comment (kbrandl) [reply docs/includes/quickstart-vs-prerequisites.md]: Fixed.

File: docs/publish/centralized-deployment.md (hunk 3 above)
Comment (kbrandl) [reply docs/publish/centralized-deployment.md]: Fixed.

File: docs/develop/convert-javascript-to-typescript.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ This article shows you how to create an Excel add-in using Visual Studio and the
     > [!NOTE]
     > TypeScript should be installed by default with Visual Studio 2017, but you can [use the Visual Studio Installer](https://docs.microsoft.com/visualstudio/install/modify-visual-studio) to confirm that it is installed. In the Visual Studio Installer, select the **Individual components** tab and then verify that **TypeScript 2.3 SDK** is selected under **SDKs, libraries, and frameworks**.
 
-- Excel 2016
+- Excel
```
Comment (kbrandl) [inline docs/develop/convert-javascript-to-typescript.md]: Changed to "Excel 2016 or later"

File: docs/excel/excel-add-ins-advanced-concepts.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ ms.date: 1/18/2018
 
 # Excel JavaScript API advanced concepts
 
-This article builds upon the information in [Excel JavaScript API core concepts](excel-add-ins-core-concepts.md) to describe some of the more advanced concepts that are essential to building complex add-ins for Excel 2016. 
+This article builds upon the information in [Excel JavaScript API core concepts](excel-add-ins-core-concepts.md) to describe some of the more advanced concepts that are essential to building complex add-ins for Excel.
```
Comment (kbrandl) [inline docs/excel/excel-add-ins-advanced-concepts.md]: Changed to "2016 or later"

File: docs/excel/excel-add-ins-advanced-concepts.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ An Excel add-in interacts with objects in Excel by using the JavaScript API for
 
 * **Common APIs**: Introduced with Office 2013, the common APIs (also referred to as the [Shared API](https://dev.office.com/reference/add-ins/javascript-api-for-office)) can be used to access features such as UI, dialogs, and client settings that are common across multiple types of host applications such as Word, Excel, and PowerPoint.
 
-While you'll likely use the Excel JavaScript API to develop the majority of functionality in add-ins that target Excel 2016, you'll also use objects in the Shared API. For example:
+While you'll likely use the Excel JavaScript API to develop the majority of functionality in add-ins that target Excel, you'll also use objects in the Shared API. For example:
```
Comment (kbrandl) [inline docs/excel/excel-add-ins-advanced-concepts.md]: Changed to "2016 or later"

File: docs/excel/excel-add-ins-core-concepts.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
-description: ''
+description: Use the Excel JavaScript API to build add-ins for Excel.
 ms.date: 12/04/2017
 ---
 
 
 # Excel JavaScript API core concepts
  
-This article describes how to use the [Excel JavaScript API](https://dev.office.com/reference/add-ins/excel/excel-add-ins-reference-overview) to build add-ins for Excel 2016. It introduces core concepts that are fundamental to using the API and provides guidance for performing specific tasks such as reading or writing to a large range, updating all cells in range, and more.
+This article describes how to use the [Excel JavaScript API](https://dev.office.com/reference/add-ins/excel/excel-add-ins-reference-overview) to build add-ins for Excel. It introduces core concepts that are fundamental to using the API and provides guidance for performing specific tasks such as reading or writing to a large range, updating all cells in range, and more.
```
Comment (kbrandl) [inline docs/excel/excel-add-ins-core-concepts.md]: "Changed to "Excel 2016 or later"

**Pull request merged**